docs/topics/native/native-target-support.md

```diff
@@ -71,12 +74,32 @@ Mind the following terms used in tier tables:
 > 
 {style="note"}
 
-## For library authors
+### For library authors
 
 We don't recommend library authors to test more targets or provide stricter guarantees than the Kotlin/Native compiler
 does. You can use the following approach when considering support for native targets:
 
 * Support all the targets from tiers 1, 2, and 3.
 * Regularly test targets from tiers 1 and 2 that support running tests out of the box.
 
-The Kotlin team uses this approach in the official Kotlin libraries, for example, [kotlinx.coroutines](coroutines-guide.md) and [kotlinx.serialization](serialization.md).
\ No newline at end of file
+The Kotlin team uses this approach in the official Kotlin libraries, for example, [kotlinx.coroutines](coroutines-guide.md) and [kotlinx.serialization](serialization.md).
+
+## Hosts
+
+The Kotlin/Native compiler supports the following hosts:
+
+| Host OS                   | Architecture     | Targets                                               |
+|---------------------------|------------------|-------------------------------------------------------|
+| macOS (Apple Silicon)     | ARM64 platforms  | `macosArm64`, `iosArm64`, `tvosArm64`, `watchosArm64` |
+| macOS (Intel chips)       | x86_64 platforms | `macosX64`, `iosX64`, `tvosX64`, `watchosX64`         |
+| Linux                     | x86_64 platforms | `linuxX64`                                            |
+| Windows (MinGW toolchain) | x86_64 platforms | `mingwX64`                                            |
```

Comment:
On Linux and Windows we can compile all possible non-Apple targets, even with cinterops. With cross-compilation we can also produce .klib's from every supported host (the only restriction is custom cinterops on Apple targets)